src/components/EmojiStaticKeyboard.tsx

```diff
@@ -60,6 +66,16 @@ export const EmojiStaticKeyboard = React.memo(
     }, [activeCategoryIndex, enableCategoryChangeAnimation, shouldAnimateScroll])
 
     const keyExtractor = React.useCallback((item: EmojisByCategory) => item.title, [])
+    const scrollNav = React.useRef(new Animated.Value(0)).current
+
+    const handleScroll = React.useCallback(
+      (el: NativeSyntheticEvent<NativeScrollEvent>) => {
+        const index = el.nativeEvent.contentOffset.x / width
+        scrollNav.setValue(index * CATEGORY_ELEMENT_WIDTH)
```

Comment:
I've just changed Spring animation to setValue because now animation feel lot more responsive @mateki0